robot_driver/include/robot_driver/estimators/ekf_filter_estimator.h

```diff
@@ -0,0 +1,32 @@
+#ifndef EKF_FILTER_H
+#define EKF_FILTER_H
```

Comment:
Isn't EKF Filter redundant? Extended Kalman Filter Filter. I think this can just be ekf_estimator.